algebird-core/src/main/scala/com/twitter/algebird/MinHasher.scala

```diff
@@ -88,6 +88,16 @@ abstract class MinHasher[H](val numHashes: Int, val numBands: Int)(implicit n: N
     buildArray(left.bytes, right.bytes){ (l, r) => if (l == r) n.one else n.zero }
       .map{ _.toDouble }.sum / numHashes
 
+  /**
+   * Generalized Jaccard similarity estimation for multiple sets (size of intersection / size of union).
+   * Jsim(S1..Sn) = P(hmin1 == hmin2 == ... == hminn) / numHashes
+   */
+  def similarityMulti(sigs: MinHashSignature*): Double = {
```

Comment:
can you put this on the companion object? that should preserve binary compatibility and we can release it in the 0.13 branch.